jsonschema/AcqSvProfile.json

```diff
@@ -13,7 +13,7 @@
   "$schema": "http://json-schema.org/draft-06/schema#",
   "$id": "#AcqSvProfile",
   "title":"AcqSvProfile",
-  "description":"Profile for a specific SV for debugging purposes,\nThe message describes SV profile during acquisition time.,\nThe message is used to debug and measure the performance.\n",
+  "description":"Profile for a specific SV for debugging purposes The message describes SV profile during acquisition time. The message is used to debug and measure the performance.\n",
```

Comment:
dropping `\n` here.  not sure whether this is good or bad.

Reply:
Don't think this matters, the description field should be ignored by jsonschema tools, it's maintained here for documentation purposes

Reply:
The first segment comes from yaml `short_desc` and is almost never terminated with a `.`, but ~40 messages have a trailing `.`.  So a `.` needs to be inserted here, and the trailing `.` needs to be removed from those ~40 messages (and example yaml explicitly state no trailing dot).

Reply:
👍